subsys/net/lib/azure_fota/azure_fota.c

```diff
@@ -82,10 +108,51 @@ static char current_version[CONFIG_AZURE_FOTA_VERSION_MAX_LEN] =
 BUILD_ASSERT(0, "CONFIG_AZURE_FOTA_SEC_TAG must be defined");
 #endif
 
```

Comment:
Maybe add comments to signify where the static and internal interface begins? In the spirit of the comment present above that introduces the static variable section. This is out of scope I guess.